inc/rccl/rccl-version.h.in

```diff
@@ -0,0 +1,17 @@
+/* ************************************************************************
+ * Copyright 2016 Advanced Micro Devices, Inc.
+ * ************************************************************************ */
+
+/* the configured version and settings
+ */
+#ifndef RCCL_VERSION_H_
+#define RCCL_VERSION_H_
+
+// clang-format off
+#define RCCL_VERSION_MAJOR @rccl_VERSION_MAJOR@
```

Comment:
Just curious: where does it get the value of `@rccl_VERSION_MAJOR@` from?

Reply:
In CMakeLists.txt. there is version:
rocm_setup_version(
    VERSION 0.6.0
)

This version is already used in Debian package version, i.e. rccl-0.6.0-9a39372-dirty-Linux.deb

Reply:
Can you please update Makefile too?

Reply:
Is the Makefile actually used?  I thought CMake was preferred?

Reply:
I understand this Makefile is a shortcut for quick testing... But I agree we should either fix it to at least working or just remove it.
It will take a lot of scripting in Makefile to manually parsing CMakeLists.txt to get correct versions. But I can put a hack in Makefile to fix version number as 0.0.1 for this special case. Is that OK?

Reply:
Added Makefile change so make can work there.